packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -107,6 +107,21 @@ const someStage = pipeline.addStage({
 });
 ```
 
+You can disable transition to a Stage:
+
+```ts
+// Disable transition to a stage
+declare const pipeline: codepipeline.Pipeline;
+
+const someStage = pipeline.addStage({
+  stageName: 'SomeStage',
+  disableTransition: true,
```

Comment:
Actually, how about we flip the logic completely? Instead of "disable" being `false` by default, let's make it `transitionToEnabled`, and `true` by default, and you can make it `false` if you want to.

Thoughts on this?

Reply:
Yes, better.